### PR TITLE
[XEDRA Evolved] Investigation -> deduction

### PR DESCRIPTION
**Pull request by GuardianDll**
#### Summary
None

#### Purpose of change

All skills in the game starts from small letter, except the investigation

#### Describe the solution

Replace upper case wilt lower case
Also replace the whole name 🤷 

#### Additional context

One small step for man..

### CODE REVIEW

**Comment by TheShadowFerret** [comment]
Why is this skill called investigation when its id is deduction 🤔

**Comment by GuardianDll** [comment]
> Why is this skill called investigation when its id is deduction 🤔

@Maleclypse?

**Comment by Maleclypse** [comment]
> > Why is this skill called investigation when its id is deduction 🤔
> 
> @Maleclypse?

I couldn’t decide which I liked better and it probably makes more sense as deduction. Especially since it’s used for spells in this mod currently. I’d originally planned to have two skills but the mod isn’t big enough for that yet.

File: data/mods/Xedra_Evolved/skills.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   {
     "type": "skill",
     "ident": "deduction",
-    "name": { "str": "Investigation" },
+    "name": { "str": "investigation" },
```
Comment (Maleclypse) [inline data/mods/Xedra_Evolved/skills.json:5]:
```suggestion
    "name": { "str": "deduction" },
```
Let's just go ahead and make the change based on the discussion below.

**Review by Maleclypse — APPROVED** [review]

**Pull request merged**